internal/pkg/cli/env_delete.go

```diff
@@ -38,6 +37,10 @@ const (
 	fmtDeleteEnvComplete = "Deleted environment %s from project %s."
 )
 
+type resourceGetter interface {
+	GetResources(*resourcegroupstaggingapi.GetResourcesInput) (*resourcegroupstaggingapi.GetResourcesOutput, error)
+}
```

Comment:
I wonder if we should group all these interfaces into a single file like `iface.go` to make it easier to discover them. I'm not sure if that's a common thing though :/

Reply:
Yeah I agree :+1: I'll cut an issue on that and also remove all the mock for iface in case the automatic bump fails again.